enterprise/coderd/license/license.go

```diff
@@ -100,6 +100,13 @@ func LicensesEntitlements(
 	//   'Entitlements' group as a whole.
 	for _, license := range licenses {
 		claims, err := ParseClaims(license.JWT, keys)
+		var vErr *jwt.ValidationError
+		if xerrors.As(err, &vErr) && vErr.Is(jwt.ErrTokenNotValidYet) {
+			// The license isn't valid yet.  We don't consider any entitlements contained in it, but
+			// it's also not an error.  Just skip it silently.  This can happen if an administrator
+			// uploads a license for a new term that hasn't started yet.
+			continue
+		}
```

Comment:
Why not just use `ParseClaimsIgnoreNbf(license.JWT, keys)` instead of `ParseClaims`

Reply:
Because in this context, the license is still invalid: we don't want to process entitlements for terms that haven't started yet. So, I want the Go `error`, but I don't want to log it. `ParseClaimsIngoreNbf` doesn't return an error if `nbf` is violated.

Reply:
Ah, makes sense. Feel like we should try to reuse this logic somehow.